### ISSUE TITLE
Upcoming Events

### ISSUE THREAD

**Issue by zesterer** [cssbristol/freshers-booklet-2017#2]
Apologies for such an abuse of GitHub's issue system, but I was unable to find the contact details of the committee and - for my sins - do not use Facebook. This seemed like the most reasonable way of making contact. Are there details of upcoming society events available online? I have so far been unable to find any on the SU page.

**Comment by rossng** [comment]
Yes! The [CSS Bristol](https://www.cssbristol.co.uk/) website has all this stuff.

Also see:
* https://cssbristol.co.uk/contact/
* https://cssbristol.co.uk/pages/freshers/
* https://cssbristol.co.uk/newsletter/
